arcade/gui/widgets/text.py

```diff
@@ -195,6 +219,11 @@ def text(self, value):
 
     @property
     def multiline(self):
+        """
+        Multiline state of the widget. Newlines indicated with ``\n`` will only
+        be honored if this is set to true. If you want a scrollable text widget,
+        use :py:class:`~arcade.gui.TextArea`.
+        """
```

Comment:
```suggestion
        """
        Get or set the multiline mode.

        Newline characters (``"\\n"``) will only be honored when this is set to ``True``.
        If you want a scrollable text widget, please use :py:class:`~arcade.gui.UITextArea`
        instead.
        """
```
1. The unescaped `\n` is the root of the error sphinx is giving you, but it's not reporting it in the right place due to a bug in our current version
2. `TextArea` does not exist, but `UITextArea` does.
3. The summary line should be separate from the rest of the docstring as shown above.